sdk-lib/src/main/java/cash/z/ecc/android/sdk/Synchronizer.kt

```diff
@@ -132,14 +135,28 @@ interface Synchronizer {
     val accountsFlow: Flow<List<Account>?>
 
     /**
-     * Measure connection quality and speed of given [servers].
+     * Tells the wallet to track an account using a unified full viewing key.
      *
-     * @return a [Flow] of fastest servers which updates it's state during measurement stages
+     * Returns details about the imported account, including the unique account identifier for
+     * the newly-created wallet database entry. Unlike the other account creation APIs, no spending key is returned
+     * because the wallet has no information about how the UFVK was derived.
+     *
+     * @param purpose Metadata describing whether or not data required for spending should be tracked by the wallet
+     * @param recoverUntil An optional height at which the wallet should exit "recovery mode"
+     * @param setup The account's setup information. See [AccountImportSetup] for more.
+     * @param treeState The tree state corresponding to the last block prior to the wallet's birthday height
+     *
+     * @return Account containing details about the imported account, including the unique account identifier for the
+     * newly-created wallet database entry
+     *
+     * @throws [InitializeException.ImportAccountException] in case of the operation failure
      */
-    suspend fun getFastestServers(
-        context: Context,
-        servers: List<LightWalletEndpoint>
-    ): Flow<FastestServersResult>
+    suspend fun importAccountByUfvk(
+        purpose: AccountPurpose,
+        recoverUntil: Long?,
```

Comment:
where do we get recoverUntil and treeState from ?

Reply:
It's the same as in the case of the `createAccount` API. Check its usage, please.